client/test/build.gradle

```diff
@@ -50,6 +50,7 @@ jarHell.enabled = false
 // TODO: should we have licenses for our test deps?
 tasks.named("dependencyLicenses").configure { it.enabled = false }
 dependenciesInfo.enabled = false
+dependenciesGraph.enabled = false
```

Comment:
please use the task avoidance api here: tasks.named("dependenciesGraph").configure { it.enabled = false }